src/Delegation.sol

```diff
@@ -204,6 +219,9 @@ contract Delegation is IDelegation, EIP712, GuardedExecutor {
     /// This constant is a pun for "chain ID 0".
     uint16 public constant MULTICHAIN_NONCE_PREFIX = 0xc1d0;
 
+    /// @dev Time period after which the contract can be unpaused by anyone.
+    uint256 public constant PAUSE_EXPIRY = 4 weeks;
```

Comment:
`PAUSE_TIMEOUT`? 

Somehow `PAUSE_EXPIRY` feels like it's the exact timestamp when people can unpause.